[2025-09-13T09:46:13.202+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-13T09:46:13.255+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_intermediate.intermediate.locations.test manual__2025-09-13T09:45:02.800484+00:00 [queued]>
[2025-09-13T09:46:13.271+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_intermediate.intermediate.locations.test manual__2025-09-13T09:45:02.800484+00:00 [queued]>
[2025-09-13T09:46:13.281+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-13T09:46:13.308+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtTestVirtualenvOperator): intermediate.locations.test> on 2025-09-13 09:45:02.800484+00:00
[2025-09-13T09:46:13.320+0000] {standard_task_runner.py:64} INFO - Started process 7279 to run task
[2025-09-13T09:46:13.325+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_intermediate', 'intermediate.locations.test', 'manual__2025-09-13T09:45:02.800484+00:00', '--job-id', '239', '--raw', '--subdir', 'DAGS_FOLDER/dbt_intermediate.py', '--cfg-path', '/tmp/tmp2sartzh7']
[2025-09-13T09:46:13.330+0000] {standard_task_runner.py:91} INFO - Job 239: Subtask intermediate.locations.test
[2025-09-13T09:46:13.550+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_intermediate.intermediate.locations.test manual__2025-09-13T09:45:02.800484+00:00 [running]> on host e95bde6b8d07
[2025-09-13T09:46:13.799+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_intermediate' AIRFLOW_CTX_TASK_ID='intermediate.locations.test' AIRFLOW_CTX_EXECUTION_DATE='2025-09-13T09:45:02.800484+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-13T09:45:02.800484+00:00'
[2025-09-13T09:46:13.801+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-13T09:46:13.814+0000] {baseoperator.py:400} WARNING - DbtTestVirtualenvOperator.execute cannot be called outside TaskInstance!
[2025-09-13T09:46:13.815+0000] {virtualenv.py:120} INFO - Checking if the virtualenv lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock exists
[2025-09-13T09:46:13.816+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:13.817+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:13.817+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:14.819+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:14.820+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:14.820+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:15.822+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:15.823+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:15.824+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:16.825+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:16.826+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:16.827+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:17.828+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:17.829+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:17.830+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:18.831+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:18.832+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:18.833+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:19.835+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:19.836+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:19.837+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:20.838+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:20.840+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:20.840+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:21.842+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:21.843+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:21.843+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:22.845+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:22.846+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:22.846+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:23.848+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:23.849+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:23.849+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:24.851+0000] {virtualenv.py:178} INFO - Checking for running process with PID 7037
[2025-09-13T09:46:24.852+0000] {virtualenv.py:181} INFO - Process 7037 running: True and has the lock /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock.
[2025-09-13T09:46:24.853+0000] {virtualenv.py:122} INFO - Waiting for virtualenv lock to be released
[2025-09-13T09:46:25.854+0000] {virtualenv.py:126} INFO - Acquiring the virtualenv lock
[2025-09-13T09:46:25.855+0000] {virtualenv.py:195} INFO - Acquiring lock at /tmp/cosmos_venv_ecom/cosmos_virtualenv.lock with pid 7279
[2025-09-13T09:46:25.855+0000] {virtualenv.py:153} INFO - Creating or updating the virtualenv at `/tmp/cosmos_venv_ecom
[2025-09-13T09:46:25.856+0000] {process_utils.py:183} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/cosmos_venv_ecom --python=python3
[2025-09-13T09:46:25.856+0000] {process_utils.py:187} INFO - Output:
[2025-09-13T09:46:27.099+0000] {process_utils.py:191} INFO - created virtual environment CPython3.10.14.final.0-64 in 485ms
[2025-09-13T09:46:27.100+0000] {process_utils.py:191} INFO -   creator CPython3Posix(dest=/tmp/cosmos_venv_ecom, clear=False, no_vcs_ignore=False, global=False)
[2025-09-13T09:46:27.100+0000] {process_utils.py:191} INFO -   seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2025-09-13T09:46:27.101+0000] {process_utils.py:191} INFO -     added seed packages: MarkupSafe==3.0.2, PyYAML==6.0.2, agate==1.9.1, annotated_types==0.7.0, attrs==25.3.0, babel==2.17.0, certifi==2025.8.3, charset_normalizer==3.4.3, click==8.2.1, clickhouse_connect==0.8.18, clickhouse_driver==0.2.9, colorama==0.4.6, daff==1.4.2, dbt_adapters==1.16.7, dbt_clickhouse==1.9.3, dbt_common==1.30.0, dbt_core==1.10.11, dbt_extractor==0.6.0, dbt_protos==1.0.375, dbt_semantic_interfaces==0.9.0, deepdiff==8.6.1, idna==3.10, importlib_metadata==8.7.0, isodate==0.6.1, jinja2==3.1.6, jsonschema==4.25.1, jsonschema_specifications==2025.9.1, leather==0.4.0, lz4==4.4.4, mashumaro==3.14, more_itertools==10.8.0, msgpack==1.1.1, networkx==3.4.2, orderly_set==5.5.0, packaging==25.0, parsedatetime==2.6, pathspec==0.12.1, pip==25.2, protobuf==6.32.1, pydantic==2.11.7, pydantic_core==2.33.2, python_dateutil==2.9.0.post0, python_slugify==8.0.4, pytimeparse==1.1.8, pytz==2025.2, referencing==0.36.2, requests==2.32.5, rpds_py==0.27.1, setuptools==80.9.0, six==1.17.0, snowplow_tracker==1.1.0, sqlparse==0.5.3, text_unidecode==1.3, typing_extensions==4.15.0, typing_inspection==0.4.1, tzlocal==5.3.1, urllib3==2.5.0, wheel==0.45.1, zipp==3.23.0, zstandard==0.24.0
[2025-09-13T09:46:27.101+0000] {process_utils.py:191} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2025-09-13T09:46:27.163+0000] {process_utils.py:183} INFO - Executing cmd: /tmp/cosmos_venv_ecom/bin/pip install dbt-clickhouse
[2025-09-13T09:46:27.163+0000] {process_utils.py:187} INFO - Output:
[2025-09-13T09:46:28.446+0000] {process_utils.py:191} INFO - Requirement already satisfied: dbt-clickhouse in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (1.9.3)
[2025-09-13T09:46:28.449+0000] {process_utils.py:191} INFO - Requirement already satisfied: dbt-core>=1.9 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-clickhouse) (1.10.11)
[2025-09-13T09:46:28.450+0000] {process_utils.py:191} INFO - Requirement already satisfied: clickhouse-connect>=0.6.22 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-clickhouse) (0.8.18)
[2025-09-13T09:46:28.451+0000] {process_utils.py:191} INFO - Requirement already satisfied: clickhouse-driver>=0.2.6 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-clickhouse) (0.2.9)
[2025-09-13T09:46:28.452+0000] {process_utils.py:191} INFO - Requirement already satisfied: setuptools>=0.69 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-clickhouse) (80.9.0)
[2025-09-13T09:46:28.455+0000] {process_utils.py:191} INFO - Requirement already satisfied: certifi in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from clickhouse-connect>=0.6.22->dbt-clickhouse) (2025.8.3)
[2025-09-13T09:46:28.455+0000] {process_utils.py:191} INFO - Requirement already satisfied: urllib3>=1.26 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from clickhouse-connect>=0.6.22->dbt-clickhouse) (2.5.0)
[2025-09-13T09:46:28.456+0000] {process_utils.py:191} INFO - Requirement already satisfied: pytz in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from clickhouse-connect>=0.6.22->dbt-clickhouse) (2025.2)
[2025-09-13T09:46:28.457+0000] {process_utils.py:191} INFO - Requirement already satisfied: zstandard in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from clickhouse-connect>=0.6.22->dbt-clickhouse) (0.24.0)
[2025-09-13T09:46:28.458+0000] {process_utils.py:191} INFO - Requirement already satisfied: lz4 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from clickhouse-connect>=0.6.22->dbt-clickhouse) (4.4.4)
[2025-09-13T09:46:28.460+0000] {process_utils.py:191} INFO - Requirement already satisfied: tzlocal in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from clickhouse-driver>=0.2.6->dbt-clickhouse) (5.3.1)
[2025-09-13T09:46:28.465+0000] {process_utils.py:191} INFO - Requirement already satisfied: agate<1.10,>=1.7.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (1.9.1)
[2025-09-13T09:46:28.466+0000] {process_utils.py:191} INFO - Requirement already satisfied: Jinja2<4,>=3.1.3 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (3.1.6)
[2025-09-13T09:46:28.467+0000] {process_utils.py:191} INFO - Requirement already satisfied: mashumaro<3.15,>=3.9 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from mashumaro[msgpack]<3.15,>=3.9->dbt-core>=1.9->dbt-clickhouse) (3.14)
[2025-09-13T09:46:28.468+0000] {process_utils.py:191} INFO - Requirement already satisfied: click<9.0,>=8.0.2 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (8.2.1)
[2025-09-13T09:46:28.469+0000] {process_utils.py:191} INFO - Requirement already satisfied: jsonschema<5.0,>=4.19.1 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (4.25.1)
[2025-09-13T09:46:28.471+0000] {process_utils.py:191} INFO - Requirement already satisfied: networkx<4.0,>=2.3 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (3.4.2)
[2025-09-13T09:46:28.472+0000] {process_utils.py:191} INFO - Requirement already satisfied: protobuf<7.0,>=6.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (6.32.1)
[2025-09-13T09:46:28.473+0000] {process_utils.py:191} INFO - Requirement already satisfied: requests<3.0.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (2.32.5)
[2025-09-13T09:46:28.474+0000] {process_utils.py:191} INFO - Requirement already satisfied: snowplow-tracker<2.0,>=1.0.2 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (1.1.0)
[2025-09-13T09:46:28.475+0000] {process_utils.py:191} INFO - Requirement already satisfied: pathspec<0.13,>=0.9 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (0.12.1)
[2025-09-13T09:46:28.476+0000] {process_utils.py:191} INFO - Requirement already satisfied: sqlparse<0.6.0,>=0.5.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (0.5.3)
[2025-09-13T09:46:28.477+0000] {process_utils.py:191} INFO - Requirement already satisfied: dbt-extractor<=0.6,>=0.5.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (0.6.0)
[2025-09-13T09:46:28.478+0000] {process_utils.py:191} INFO - Requirement already satisfied: dbt-semantic-interfaces<0.10,>=0.9.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (0.9.0)
[2025-09-13T09:46:28.479+0000] {process_utils.py:191} INFO - Requirement already satisfied: dbt-common<2.0,>=1.27.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (1.30.0)
[2025-09-13T09:46:28.480+0000] {process_utils.py:191} INFO - Requirement already satisfied: dbt-adapters<2.0,>=1.15.5 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (1.16.7)
[2025-09-13T09:46:28.481+0000] {process_utils.py:191} INFO - Requirement already satisfied: dbt-protos<2.0,>=1.0.346 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (1.0.375)
[2025-09-13T09:46:28.482+0000] {process_utils.py:191} INFO - Requirement already satisfied: pydantic<3 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (2.11.7)
[2025-09-13T09:46:28.483+0000] {process_utils.py:191} INFO - Requirement already satisfied: packaging>20.9 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (25.0)
[2025-09-13T09:46:28.485+0000] {process_utils.py:191} INFO - Requirement already satisfied: pyyaml>=6.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (6.0.2)
[2025-09-13T09:46:28.486+0000] {process_utils.py:191} INFO - Requirement already satisfied: daff>=1.3.46 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (1.4.2)
[2025-09-13T09:46:28.487+0000] {process_utils.py:191} INFO - Requirement already satisfied: typing-extensions>=4.4 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-core>=1.9->dbt-clickhouse) (4.15.0)
[2025-09-13T09:46:28.491+0000] {process_utils.py:191} INFO - Requirement already satisfied: Babel>=2.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from agate<1.10,>=1.7.0->dbt-core>=1.9->dbt-clickhouse) (2.17.0)
[2025-09-13T09:46:28.493+0000] {process_utils.py:191} INFO - Requirement already satisfied: isodate>=0.5.4 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from agate<1.10,>=1.7.0->dbt-core>=1.9->dbt-clickhouse) (0.6.1)
[2025-09-13T09:46:28.494+0000] {process_utils.py:191} INFO - Requirement already satisfied: leather>=0.3.2 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from agate<1.10,>=1.7.0->dbt-core>=1.9->dbt-clickhouse) (0.4.0)
[2025-09-13T09:46:28.494+0000] {process_utils.py:191} INFO - Requirement already satisfied: parsedatetime!=2.5,>=2.1 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from agate<1.10,>=1.7.0->dbt-core>=1.9->dbt-clickhouse) (2.6)
[2025-09-13T09:46:28.495+0000] {process_utils.py:191} INFO - Requirement already satisfied: python-slugify>=1.2.1 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from agate<1.10,>=1.7.0->dbt-core>=1.9->dbt-clickhouse) (8.0.4)
[2025-09-13T09:46:28.496+0000] {process_utils.py:191} INFO - Requirement already satisfied: pytimeparse>=1.1.5 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from agate<1.10,>=1.7.0->dbt-core>=1.9->dbt-clickhouse) (1.1.8)
[2025-09-13T09:46:28.510+0000] {process_utils.py:191} INFO - Requirement already satisfied: colorama<0.5,>=0.3.9 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-common<2.0,>=1.27.0->dbt-core>=1.9->dbt-clickhouse) (0.4.6)
[2025-09-13T09:46:28.512+0000] {process_utils.py:191} INFO - Requirement already satisfied: deepdiff<9.0,>=7.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-common<2.0,>=1.27.0->dbt-core>=1.9->dbt-clickhouse) (8.6.1)
[2025-09-13T09:46:28.516+0000] {process_utils.py:191} INFO - Requirement already satisfied: python-dateutil<3.0,>=2.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-common<2.0,>=1.27.0->dbt-core>=1.9->dbt-clickhouse) (2.9.0.post0)
[2025-09-13T09:46:28.525+0000] {process_utils.py:191} INFO - Requirement already satisfied: importlib-metadata<9,>=6.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-semantic-interfaces<0.10,>=0.9.0->dbt-core>=1.9->dbt-clickhouse) (8.7.0)
[2025-09-13T09:46:28.527+0000] {process_utils.py:191} INFO - Requirement already satisfied: more-itertools<11.0,>=8.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from dbt-semantic-interfaces<0.10,>=0.9.0->dbt-core>=1.9->dbt-clickhouse) (10.8.0)
[2025-09-13T09:46:28.538+0000] {process_utils.py:191} INFO - Requirement already satisfied: orderly-set<6,>=5.4.1 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from deepdiff<9.0,>=7.0->dbt-common<2.0,>=1.27.0->dbt-core>=1.9->dbt-clickhouse) (5.5.0)
[2025-09-13T09:46:28.543+0000] {process_utils.py:191} INFO - Requirement already satisfied: zipp>=3.20 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from importlib-metadata<9,>=6.0->dbt-semantic-interfaces<0.10,>=0.9.0->dbt-core>=1.9->dbt-clickhouse) (3.23.0)
[2025-09-13T09:46:28.545+0000] {process_utils.py:191} INFO - Requirement already satisfied: six in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from isodate>=0.5.4->agate<1.10,>=1.7.0->dbt-core>=1.9->dbt-clickhouse) (1.17.0)
[2025-09-13T09:46:28.547+0000] {process_utils.py:191} INFO - Requirement already satisfied: MarkupSafe>=2.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from Jinja2<4,>=3.1.3->dbt-core>=1.9->dbt-clickhouse) (3.0.2)
[2025-09-13T09:46:28.551+0000] {process_utils.py:191} INFO - Requirement already satisfied: attrs>=22.2.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from jsonschema<5.0,>=4.19.1->dbt-core>=1.9->dbt-clickhouse) (25.3.0)
[2025-09-13T09:46:28.552+0000] {process_utils.py:191} INFO - Requirement already satisfied: jsonschema-specifications>=2023.03.6 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from jsonschema<5.0,>=4.19.1->dbt-core>=1.9->dbt-clickhouse) (2025.9.1)
[2025-09-13T09:46:28.552+0000] {process_utils.py:191} INFO - Requirement already satisfied: referencing>=0.28.4 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from jsonschema<5.0,>=4.19.1->dbt-core>=1.9->dbt-clickhouse) (0.36.2)
[2025-09-13T09:46:28.553+0000] {process_utils.py:191} INFO - Requirement already satisfied: rpds-py>=0.7.1 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from jsonschema<5.0,>=4.19.1->dbt-core>=1.9->dbt-clickhouse) (0.27.1)
[2025-09-13T09:46:28.563+0000] {process_utils.py:191} INFO - Requirement already satisfied: msgpack>=0.5.6 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from mashumaro[msgpack]<3.15,>=3.9->dbt-core>=1.9->dbt-clickhouse) (1.1.1)
[2025-09-13T09:46:28.581+0000] {process_utils.py:191} INFO - Requirement already satisfied: annotated-types>=0.6.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from pydantic<3->dbt-core>=1.9->dbt-clickhouse) (0.7.0)
[2025-09-13T09:46:28.582+0000] {process_utils.py:191} INFO - Requirement already satisfied: pydantic-core==2.33.2 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from pydantic<3->dbt-core>=1.9->dbt-clickhouse) (2.33.2)
[2025-09-13T09:46:28.584+0000] {process_utils.py:191} INFO - Requirement already satisfied: typing-inspection>=0.4.0 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from pydantic<3->dbt-core>=1.9->dbt-clickhouse) (0.4.1)
[2025-09-13T09:46:28.595+0000] {process_utils.py:191} INFO - Requirement already satisfied: charset_normalizer<4,>=2 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from requests<3.0.0->dbt-core>=1.9->dbt-clickhouse) (3.4.3)
[2025-09-13T09:46:28.596+0000] {process_utils.py:191} INFO - Requirement already satisfied: idna<4,>=2.5 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from requests<3.0.0->dbt-core>=1.9->dbt-clickhouse) (3.10)
[2025-09-13T09:46:28.634+0000] {process_utils.py:191} INFO - Requirement already satisfied: text-unidecode>=1.3 in /tmp/cosmos_venv_ecom/lib/python3.10/site-packages (from python-slugify>=1.2.1->agate<1.10,>=1.7.0->dbt-core>=1.9->dbt-clickhouse) (1.3)
[2025-09-13T09:46:28.807+0000] {local.py:470} INFO - Cloning project to writable temp directory /tmp/tmpd9iirw8g from /opt/airflow/dags/dbt/ecom_intermediate
[2025-09-13T09:46:28.821+0000] {local.py:490} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dbt_intermediate__intermediate/target/partial_parse.msgpack
[2025-09-13T09:46:28.836+0000] {config.py:362} INFO - Using user-supplied profiles.yml at /opt/airflow/dags/dbt/profiles.yml
[2025-09-13T09:46:28.848+0000] {virtualenv.py:99} INFO - Using Python binary from virtualenv: /tmp/cosmos_venv_ecom/bin/python
[2025-09-13T09:46:28.849+0000] {local.py:434} INFO - Trying to run the command:
 ['/tmp/cosmos_venv_ecom/bin/dbt', 'deps', '--project-dir', '/tmp/tmpd9iirw8g', '--profiles-dir', '/opt/***/dags/dbt', '--profile', 'ecom_intermediate', '--target', 'dev']
From /tmp/tmpd9iirw8g
[2025-09-13T09:46:28.849+0000] {subprocess.py:56} INFO - Tmp dir root location: 
 /tmp
[2025-09-13T09:46:28.850+0000] {subprocess.py:69} INFO - Running command: ['/tmp/cosmos_venv_ecom/bin/dbt', 'deps', '--project-dir', '/tmp/tmpd9iirw8g', '--profiles-dir', '/opt/***/dags/dbt', '--profile', 'ecom_intermediate', '--target', 'dev']
[2025-09-13T09:46:28.868+0000] {subprocess.py:80} INFO - Command output:
[2025-09-13T09:46:30.981+0000] {subprocess.py:90} INFO - [0m09:46:30  Running with dbt=1.10.11
[2025-09-13T09:46:31.595+0000] {subprocess.py:90} INFO - [0m09:46:31  Installing dbt-labs/dbt_utils
[2025-09-13T09:46:32.140+0000] {subprocess.py:90} INFO - [0m09:46:32  Installed from version 1.3.0
[2025-09-13T09:46:32.142+0000] {subprocess.py:90} INFO - [0m09:46:32  Updated version available: 1.3.1
[2025-09-13T09:46:32.142+0000] {subprocess.py:90} INFO - [0m09:46:32
[2025-09-13T09:46:32.143+0000] {subprocess.py:90} INFO - [0m09:46:32  Updates available for packages: ['dbt-labs/dbt_utils']
[2025-09-13T09:46:32.143+0000] {subprocess.py:90} INFO - Update your versions in packages.yml, then run dbt deps
[2025-09-13T09:46:33.918+0000] {subprocess.py:94} INFO - Command exited with return code 0
[2025-09-13T09:46:33.918+0000] {local.py:441} INFO - Update your versions in packages.yml, then run dbt deps
[2025-09-13T09:46:33.919+0000] {virtualenv.py:99} INFO - Using Python binary from virtualenv: /tmp/cosmos_venv_ecom/bin/python
[2025-09-13T09:46:33.919+0000] {local.py:434} INFO - Trying to run the command:
 ['/tmp/cosmos_venv_ecom/bin/dbt', 'test', '--models', 'locations', '--project-dir', '/tmp/tmpd9iirw8g', '--profiles-dir', '/opt/***/dags/dbt', '--profile', 'ecom_intermediate', '--target', 'dev']
From /tmp/tmpd9iirw8g
[2025-09-13T09:46:33.920+0000] {subprocess.py:56} INFO - Tmp dir root location: 
 /tmp
[2025-09-13T09:46:33.920+0000] {subprocess.py:69} INFO - Running command: ['/tmp/cosmos_venv_ecom/bin/dbt', 'test', '--models', 'locations', '--project-dir', '/tmp/tmpd9iirw8g', '--profiles-dir', '/opt/***/dags/dbt', '--profile', 'ecom_intermediate', '--target', 'dev']
[2025-09-13T09:46:33.938+0000] {subprocess.py:80} INFO - Command output:
[2025-09-13T09:46:36.228+0000] {subprocess.py:90} INFO - [0m09:46:36  Running with dbt=1.10.11
[2025-09-13T09:46:36.230+0000] {subprocess.py:90} INFO - [0m09:46:36  [[33mWARNING[0m]: Deprecated functionality
[2025-09-13T09:46:36.231+0000] {subprocess.py:90} INFO - Usage of `--models`, `--model`, and `-m` is deprecated in favor of `--select` or
[2025-09-13T09:46:36.232+0000] {subprocess.py:90} INFO - `-s`.
[2025-09-13T09:46:36.613+0000] {subprocess.py:90} INFO - [0m09:46:36  Registered adapter: clickhouse=1.9.3
[2025-09-13T09:46:37.646+0000] {subprocess.py:90} INFO - [0m09:46:37  Found 15 models, 50 data tests, 8 sources, 597 macros
[2025-09-13T09:46:37.650+0000] {subprocess.py:90} INFO - [0m09:46:37
[2025-09-13T09:46:37.651+0000] {subprocess.py:90} INFO - [0m09:46:37  Concurrency: 4 threads (target='dev')
[2025-09-13T09:46:37.652+0000] {subprocess.py:90} INFO - [0m09:46:37
[2025-09-13T09:46:37.791+0000] {subprocess.py:90} INFO - [0m09:46:37  1 of 7 START test not_null_locations_city ...................................... [RUN]
[2025-09-13T09:46:37.792+0000] {subprocess.py:90} INFO - [0m09:46:37  2 of 7 START test not_null_locations_country ................................... [RUN]
[2025-09-13T09:46:37.793+0000] {subprocess.py:90} INFO - [0m09:46:37  3 of 7 START test not_null_locations_location_id ............................... [RUN]
[2025-09-13T09:46:37.794+0000] {subprocess.py:90} INFO - [0m09:46:37  4 of 7 START test not_null_locations_state ..................................... [RUN]
[2025-09-13T09:46:37.894+0000] {subprocess.py:90} INFO - [0m09:46:37  1 of 7 PASS not_null_locations_city ............................................ [[32mPASS[0m in 0.10s]
[2025-09-13T09:46:37.901+0000] {subprocess.py:90} INFO - [0m09:46:37  5 of 7 START test relationships_addresses_location_id__location_id__ref_locations_  [RUN]
[2025-09-13T09:46:37.946+0000] {subprocess.py:90} INFO - [0m09:46:37  5 of 7 ERROR relationships_addresses_location_id__location_id__ref_locations_ .. [[31mERROR[0m in 0.04s]
[2025-09-13T09:46:37.952+0000] {subprocess.py:90} INFO - [0m09:46:37  6 of 7 START test relationships_customers_enriched_location_id__location_id__ref_locations_  [RUN]
[2025-09-13T09:46:37.986+0000] {subprocess.py:90} INFO - [0m09:46:37  6 of 7 ERROR relationships_customers_enriched_location_id__location_id__ref_locations_  [[31mERROR[0m in 0.03s]
[2025-09-13T09:46:37.992+0000] {subprocess.py:90} INFO - [0m09:46:37  7 of 7 START test unique_locations_location_id ................................. [RUN]
[2025-09-13T09:46:37.995+0000] {subprocess.py:90} INFO - [0m09:46:37  2 of 7 PASS not_null_locations_country ......................................... [[32mPASS[0m in 0.20s]
[2025-09-13T09:46:37.997+0000] {subprocess.py:90} INFO - [0m09:46:37  4 of 7 PASS not_null_locations_state ........................................... [[32mPASS[0m in 0.20s]
[2025-09-13T09:46:38.000+0000] {subprocess.py:90} INFO - [0m09:46:38  3 of 7 PASS not_null_locations_location_id ..................................... [[32mPASS[0m in 0.20s]
[2025-09-13T09:46:38.028+0000] {subprocess.py:90} INFO - [0m09:46:38  7 of 7 PASS unique_locations_location_id ....................................... [[32mPASS[0m in 0.03s]
[2025-09-13T09:46:38.036+0000] {subprocess.py:90} INFO - [0m09:46:38
[2025-09-13T09:46:38.038+0000] {subprocess.py:90} INFO - [0m09:46:38  Finished running 7 data tests in 0 hours 0 minutes and 0.38 seconds (0.38s).
[2025-09-13T09:46:38.119+0000] {subprocess.py:90} INFO - [0m09:46:38
[2025-09-13T09:46:38.120+0000] {subprocess.py:90} INFO - [0m09:46:38  [31mCompleted with 2 errors, 0 partial successes, and 0 warnings:[0m
[2025-09-13T09:46:38.120+0000] {subprocess.py:90} INFO - [0m09:46:38
[2025-09-13T09:46:38.121+0000] {subprocess.py:90} INFO - [0m09:46:38  [31mFailure in test relationships_addresses_location_id__location_id__ref_locations_ (models/ecom_intermediate/schema.yml)[0m
[2025-09-13T09:46:38.122+0000] {subprocess.py:90} INFO - [0m09:46:38    Database Error in test relationships_addresses_location_id__location_id__ref_locations_ (models/ecom_intermediate/schema.yml)
[2025-09-13T09:46:38.122+0000] {subprocess.py:90} INFO -   HTTPDriver for http://clickhouse:8123 received ClickHouse error code 60
[2025-09-13T09:46:38.123+0000] {subprocess.py:90} INFO -    Code: 60. DB::Exception: Unknown table expression identifier 'ecom_db_intermediate.addresses' in scope child AS (SELECT location_id AS from_field FROM ecom_db_intermediate.addresses WHERE location_id IS NOT NULL). (UNKNOWN_TABLE) (version 24.8.14.39 (official build))
[2025-09-13T09:46:38.124+0000] {subprocess.py:90} INFO -   compiled code at target/run/ecom_intermediate/models/ecom_intermediate/schema.yml/relationships_addresses_eeba738b799cfad2ecf131432556b9a6.sql
[2025-09-13T09:46:38.124+0000] {subprocess.py:90} INFO - [0m09:46:38
[2025-09-13T09:46:38.125+0000] {subprocess.py:90} INFO - [0m09:46:38    compiled code at target/compiled/ecom_intermediate/models/ecom_intermediate/schema.yml/relationships_addresses_eeba738b799cfad2ecf131432556b9a6.sql
[2025-09-13T09:46:38.125+0000] {subprocess.py:90} INFO - [0m09:46:38
[2025-09-13T09:46:38.125+0000] {subprocess.py:90} INFO - [0m09:46:38  [31mFailure in test relationships_customers_enriched_location_id__location_id__ref_locations_ (models/ecom_intermediate/schema.yml)[0m
[2025-09-13T09:46:38.126+0000] {subprocess.py:90} INFO - [0m09:46:38    Database Error in test relationships_customers_enriched_location_id__location_id__ref_locations_ (models/ecom_intermediate/schema.yml)
[2025-09-13T09:46:38.126+0000] {subprocess.py:90} INFO -   HTTPDriver for http://clickhouse:8123 received ClickHouse error code 60
[2025-09-13T09:46:38.127+0000] {subprocess.py:90} INFO -    Code: 60. DB::Exception: Unknown table expression identifier 'ecom_db_intermediate.customers_enriched' in scope child AS (SELECT location_id AS from_field FROM ecom_db_intermediate.customers_enriched WHERE location_id IS NOT NULL). (UNKNOWN_TABLE) (version 24.8.14.39 (official build))
[2025-09-13T09:46:38.127+0000] {subprocess.py:90} INFO -   compiled code at target/run/ecom_intermediate/models/ecom_intermediate/schema.yml/relationships_customers_enrich_d5148ed200b789295089897165c05076.sql
[2025-09-13T09:46:38.128+0000] {subprocess.py:90} INFO - [0m09:46:38
[2025-09-13T09:46:38.128+0000] {subprocess.py:90} INFO - [0m09:46:38    compiled code at target/compiled/ecom_intermediate/models/ecom_intermediate/schema.yml/relationships_customers_enrich_d5148ed200b789295089897165c05076.sql
[2025-09-13T09:46:38.128+0000] {subprocess.py:90} INFO - [0m09:46:38
[2025-09-13T09:46:38.129+0000] {subprocess.py:90} INFO - [0m09:46:38  Done. PASS=5 WARN=0 ERROR=2 SKIP=0 NO-OP=0 TOTAL=7
[2025-09-13T09:46:38.130+0000] {subprocess.py:90} INFO - [0m09:46:38  [[33mWARNING[0m][DeprecationsSummary]: Deprecated functionality
[2025-09-13T09:46:38.131+0000] {subprocess.py:90} INFO - Summary of encountered deprecations:
[2025-09-13T09:46:38.132+0000] {subprocess.py:90} INFO - - ModelParamUsageDeprecation: 1 occurrence
[2025-09-13T09:46:38.132+0000] {subprocess.py:90} INFO - To see all deprecation instances instead of just the first occurrence of each,
[2025-09-13T09:46:38.133+0000] {subprocess.py:90} INFO - run command again with the `--show-all-deprecations` flag. You may also need to
[2025-09-13T09:46:38.133+0000] {subprocess.py:90} INFO - run with `--no-partial-parse` as some deprecations are only encountered during
[2025-09-13T09:46:38.134+0000] {subprocess.py:90} INFO - parsing.
[2025-09-13T09:46:39.921+0000] {subprocess.py:94} INFO - Command exited with return code 1
[2025-09-13T09:46:39.922+0000] {local.py:441} INFO - parsing.
[2025-09-13T09:46:39.939+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v12.json is above dbt-ol supported version 7. This might cause errors.
[2025-09-13T09:46:39.940+0000] {local.py:196} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/run-results/v6.json is above dbt-ol supported version 5. This might cause errors.
[2025-09-13T09:46:39.950+0000] {local.py:547} INFO - Inlets: []
[2025-09-13T09:46:39.951+0000] {local.py:548} INFO - Outlets: []
[2025-09-13T09:46:39.952+0000] {local.py:772} INFO - Assigning inlets/outlets without DatasetAlias
[2025-09-13T09:46:39.952+0000] {dag.py:3089} INFO - Sync 1 DAGs
[2025-09-13T09:46:40.052+0000] {dag.py:3947} INFO - Setting next_dagrun for dbt_intermediate to 2025-09-13 00:00:00+00:00, run_after=2025-09-14 00:00:00+00:00
[2025-09-13T09:46:40.131+0000] {local.py:253} ERROR - [0m09:46:36  Running with dbt=1.10.11
[0m09:46:36  [[33mWARNING[0m]: Deprecated functionality
Usage of `--models`, `--model`, and `-m` is deprecated in favor of `--select` or
`-s`.
[0m09:46:36  Registered adapter: clickhouse=1.9.3
[0m09:46:37  Found 15 models, 50 data tests, 8 sources, 597 macros
[0m09:46:37
[0m09:46:37  Concurrency: 4 threads (target='dev')
[0m09:46:37
[0m09:46:37  1 of 7 START test not_null_locations_city ...................................... [RUN]
[0m09:46:37  2 of 7 START test not_null_locations_country ................................... [RUN]
[0m09:46:37  3 of 7 START test not_null_locations_location_id ............................... [RUN]
[0m09:46:37  4 of 7 START test not_null_locations_state ..................................... [RUN]
[0m09:46:37  1 of 7 PASS not_null_locations_city ............................................ [[32mPASS[0m in 0.10s]
[0m09:46:37  5 of 7 START test relationships_addresses_location_id__location_id__ref_locations_  [RUN]
[0m09:46:37  5 of 7 ERROR relationships_addresses_location_id__location_id__ref_locations_ .. [[31mERROR[0m in 0.04s]
[0m09:46:37  6 of 7 START test relationships_customers_enriched_location_id__location_id__ref_locations_  [RUN]
[0m09:46:37  6 of 7 ERROR relationships_customers_enriched_location_id__location_id__ref_locations_  [[31mERROR[0m in 0.03s]
[0m09:46:37  7 of 7 START test unique_locations_location_id ................................. [RUN]
[0m09:46:37  2 of 7 PASS not_null_locations_country ......................................... [[32mPASS[0m in 0.20s]
[0m09:46:37  4 of 7 PASS not_null_locations_state ........................................... [[32mPASS[0m in 0.20s]
[0m09:46:38  3 of 7 PASS not_null_locations_location_id ..................................... [[32mPASS[0m in 0.20s]
[0m09:46:38  7 of 7 PASS unique_locations_location_id ....................................... [[32mPASS[0m in 0.03s]
[0m09:46:38
[0m09:46:38  Finished running 7 data tests in 0 hours 0 minutes and 0.38 seconds (0.38s).
[0m09:46:38
[0m09:46:38  [31mCompleted with 2 errors, 0 partial successes, and 0 warnings:[0m
[0m09:46:38
[0m09:46:38  [31mFailure in test relationships_addresses_location_id__location_id__ref_locations_ (models/ecom_intermediate/schema.yml)[0m
[0m09:46:38    Database Error in test relationships_addresses_location_id__location_id__ref_locations_ (models/ecom_intermediate/schema.yml)
  HTTPDriver for http://clickhouse:8123 received ClickHouse error code 60
   Code: 60. DB::Exception: Unknown table expression identifier 'ecom_db_intermediate.addresses' in scope child AS (SELECT location_id AS from_field FROM ecom_db_intermediate.addresses WHERE location_id IS NOT NULL). (UNKNOWN_TABLE) (version 24.8.14.39 (official build))
  compiled code at target/run/ecom_intermediate/models/ecom_intermediate/schema.yml/relationships_addresses_eeba738b799cfad2ecf131432556b9a6.sql
[0m09:46:38
[0m09:46:38    compiled code at target/compiled/ecom_intermediate/models/ecom_intermediate/schema.yml/relationships_addresses_eeba738b799cfad2ecf131432556b9a6.sql
[0m09:46:38
[0m09:46:38  [31mFailure in test relationships_customers_enriched_location_id__location_id__ref_locations_ (models/ecom_intermediate/schema.yml)[0m
[0m09:46:38    Database Error in test relationships_customers_enriched_location_id__location_id__ref_locations_ (models/ecom_intermediate/schema.yml)
  HTTPDriver for http://clickhouse:8123 received ClickHouse error code 60
   Code: 60. DB::Exception: Unknown table expression identifier 'ecom_db_intermediate.customers_enriched' in scope child AS (SELECT location_id AS from_field FROM ecom_db_intermediate.customers_enriched WHERE location_id IS NOT NULL). (UNKNOWN_TABLE) (version 24.8.14.39 (official build))
  compiled code at target/run/ecom_intermediate/models/ecom_intermediate/schema.yml/relationships_customers_enrich_d5148ed200b789295089897165c05076.sql
[0m09:46:38
[0m09:46:38    compiled code at target/compiled/ecom_intermediate/models/ecom_intermediate/schema.yml/relationships_customers_enrich_d5148ed200b789295089897165c05076.sql
[0m09:46:38
[0m09:46:38  Done. PASS=5 WARN=0 ERROR=2 SKIP=0 NO-OP=0 TOTAL=7
[0m09:46:38  [[33mWARNING[0m][DeprecationsSummary]: Deprecated functionality
Summary of encountered deprecations:
- ModelParamUsageDeprecation: 1 occurrence
To see all deprecation instances instead of just the first occurrence of each,
run command again with the `--show-all-deprecations` flag. You may also need to
run with `--no-partial-parse` as some deprecations are only encountered during
parsing.
[2025-09-13T09:46:40.141+0000] {virtualenv.py:131} INFO - Releasing virtualenv lock
[2025-09-13T09:46:40.142+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-13T09:46:40.190+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/cosmos/operators/virtualenv.py", line 144, in execute
    output = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/cosmos/operators/local.py", line 1089, in execute
    result = self.build_and_run_cmd(context=context, cmd_flags=self.add_cmd_flags())
  File "/home/airflow/.local/lib/python3.10/site-packages/cosmos/operators/local.py", line 853, in build_and_run_cmd
    result = self.run_command(
  File "/home/airflow/.local/lib/python3.10/site-packages/cosmos/operators/virtualenv.py", line 129, in run_command
    return super().run_command(cmd, env, context, run_as_async=run_as_async, async_context=async_context)
  File "/home/airflow/.local/lib/python3.10/site-packages/cosmos/operators/local.py", line 650, in run_command
    self.handle_exception(result)
  File "/home/airflow/.local/lib/python3.10/site-packages/cosmos/operators/local.py", line 254, in handle_exception_subprocess
    raise AirflowException(f"dbt command failed. The command returned a non-zero exit code {result.exit_code}.")
airflow.exceptions.AirflowException: dbt command failed. The command returned a non-zero exit code 1.
[2025-09-13T09:46:40.195+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=dbt_intermediate, task_id=intermediate.locations.test, run_id=manual__2025-09-13T09:45:02.800484+00:00, execution_date=20250913T094502, start_date=20250913T094613, end_date=20250913T094640
[2025-09-13T09:46:40.234+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 239 for task intermediate.locations.test (dbt command failed. The command returned a non-zero exit code 1.; 7279)
[2025-09-13T09:46:40.286+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 1
[2025-09-13T09:46:40.390+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-13T09:46:40.408+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
